energyplus_regressions/diffs/table_diff.py

```diff
@@ -479,6 +481,8 @@ def table_diff(
         if len(table1('tr')) != len(table2('tr')) or len(table1('td')) != len(table2('td')):
             table_size_error = 1
             count_of_size_error += table_size_error
+            table_big_diff = 1
+            count_of_big_diff += table_big_diff
```

Comment:
A table that has a mismatched size (row count change, column count change, empty, etc.), will now also report a table big diff.